### ISSUE TITLE
Improve generator.html

### ISSUE THREAD

**Issue by kalwalt** [AR-js-org/AR.js#624]
**Do you want to request a *feature* or report a *bug*?**
The marker [generator](https://github.com/AR-js-org/AR.js/blob/master/three.js/examples/marker-training/examples/generator.html) need an update, It generate a pattern marker but i think some libraries can be upgraded.

**What is the current behavior?**
It correctly generate a pattern marker but as explained above we should/could improve a bit. Maybe @nickw1 we could host somewhere on https://github.com/AR-js-org/AR-js-org.github.io? 

**If the current behavior is a bug, please provide the steps to reproduce.**
No bugs, skip this.

**Please mention other relevant information such as the browser version, Operating System and Device Name**
Tested on Chrome 131.0.6778.140 (64 bit) Windows 11.
**What is the expected behavior?**
It correctly generate a pattern marker, see my comment above.
**If this is a feature request, what is motivation or use case for changing the behavior?**
The motivation, if we can, is to improve the generator app.

**Comment by nickw1** [comment]
@kalwalt yes, hosting on github.io would make sense as it's client-side code.
